app/src/componentLibrary/fieldText/fieldText.scss

```diff
@@ -29,8 +29,6 @@
 @mixin input-base {
   width: 100%;
   margin: 2px 0 0 0;
-  padding: 0;
```

Comment:
removing it you back standard browser behaviour and it break design
`input[type="text" i] {
    padding: 1px 2px;
}`